alpine/rabbitmq/3.6.x/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM nowait/erlang:latest
+MAINTAINER Nowait <devops@nowait.com>
+
+ENV RABBITMQ_VERSION=3.6.2 \
+    RABBITMQ_DEFAULT_USER=guest \
+    RABBITMQ_DEFAULT_PASS=guest \
+    RABBITMQ_ERLANG_COOKIE="secret cookie here" \
+    PATH=/opt/rabbitmq/sbin:$PATH \
+    RABBITMQ_HOME=/opt/rabbitmq \
+    RABBITMQ_PLUGINS=rabbitmq_management
+
+RUN apk add --update --no-cache ca-certificates curl tar xz su-exec util-linux && \
+    mkdir -p /opt/rabbitmq && \
+    curl -sSL https://www.rabbitmq.com/releases/rabbitmq-server/v${RABBITMQ_VERSION}/rabbitmq-server-generic-unix-${RABBITMQ_VERSION}.tar.xz | tar -xJ -C /opt/rabbitmq --strip-components 1 && \
+    rm -rf /share/**/rabbitmq*.xz && \
+    addgroup rabbitmq && \
+    adduser -SD rabbitmq -G rabbitmq -h /opt/rabbitmq && \
+    chown -R rabbitmq:rabbitmq /opt/rabbitmq && \
```

Comment:
Can this be moved to the next run section to replace the 2 chown commands there?